test/memkind_memtier_data_movement_test.cpp

```diff
@@ -24,6 +24,26 @@
 
 #include <test/proc_stat.h>
 
+std::chrono::duration<double> bench(size_t bench_num, size_t alloc_size,
+                                    void *start_addr)
+{
+    auto start = std::chrono::steady_clock::now();
+
+    long int temp = 0;
+    for (size_t i = 0; i < bench_num; i++) {
+        for (size_t j = 0; j < (alloc_size - 1) / sizeof(int); j++) {
```

Comment:
Maybe some bigger stride would be better ? E.g. :
```C
for (size_t j = 0; j < (alloc_size - 1) / sizeof(int); j+=32) {
```

For cache line size: 128 B, sizeof(int) == 4 ; 128/4 = 32

The best scenario would be the one that maximizes PMEM_ACCESS_TIME/DRAM_ACCESS_TIME ratio.